Homework-june-13/Daniel_Mourad/aws-ec2-automation.sh

```diff
@@ -0,0 +1,414 @@
+#!/bin/bash
+
+set -e
```

Comment:
will conflict with your own error handling

Reply:
The set +e used when needed so set -e will not conflict with error handling